README.md

```diff
@@ -21,7 +21,11 @@ inputs:
     description: "Default bump level if labels are not attached [major,minor,patch]. Do nothing if it's empty"
   dry_run:
     description: "Do not actually tag next version if it's true"
+  github_token:
```

Comment:
Can you remove this from README or make it clear that users don't need to fill explicitly?